apps/emqx_message_validation/src/emqx_message_validation_schema.erl

```diff
@@ -269,3 +270,23 @@ do_validate_unique_schema_checks(
     end;
 do_validate_unique_schema_checks([_Check | Rest], Seen, Duplicated) ->
     do_validate_unique_schema_checks(Rest, Seen, Duplicated).
+
+validate_unique_topics(Topics) ->
+    Grouped = maps:groups_from_list(
```

Comment:
we'll need to update README to drop OTP 24 for this change ?

Reply:
I can update it, but this is not new: several other places already use `groups_from_list` :thinking: 

e.g.: https://github.com/emqx/emqx/blob/002dc8541ba9aad78aea48b4cb336c6b1de41cf7/apps/emqx_management/src/emqx_mgmt_api_trace.erl#L567